index.md

```diff
@@ -3,7 +3,7 @@ layout: frontpage
 
 # Header texts
 headerTitle: "The Scala Programming Language"
-headerSubtitle: "Scala is a concise high-level programming language. Scala's compiler helps you avoid bugs when writing complex applications, and its fast runtime lets you build efficient, high-performance systems, with easy access to a huge ecosystem of libraries."
+headerSubtitle: "Scala combines object-orientation with functional programming into one concise, high-level language. Scala's type system helps avoid bugs in complex applications, and its JVM target lets you build high-performance systems with easy access to a huge ecosystem of libraries."
```

Comment:
For "JVM target": the page below talks about "backends", so "JVM backend" might be more consistent.  But both "target" and "backend" are compiler jargon.  "JVM support" seems like a good compromise, since it encompasses both compiler and runtime.